Resources/Prototypes/Entities/Objects/Weapons/Guns/Launchers/launchers.yml

```diff
@@ -82,3 +82,45 @@
       magState: mag
       steps: 1
       zeroVisible: true
+
+- type: entity
+  name: Multiple rocket launcher
+  parent: LauncherBase
+  id: LauncherMultipleRocket
+  description: A modified ancient rocket-propelled grenade launcher.
```

Comment:
..ancient?

Reply:
That's the actual description of the RPG, didn't touch it.